__tests__/utils/jest-global-setup.ts

```diff
@@ -30,18 +30,21 @@ module.exports = async () => {
   if (wsEndpoint) {
     return new Promise((resolve, reject) => {
       console.log(`\nRunning BrowserService ${wsEndpoint}`);
-      const process = spawn('node', ['./dist/browser-service.js']);
-      (global as any).__browserServiceProcess = process;
-      process.stdout.on('data', data => {
+      const browserServiceProcess = spawn(
+        'node',
+        ['./dist/run-browser-service.js'],
+        { env: { ...process.env, DEBUG: 'true' } }
+      );
+      (global as any).__browserServiceProcess = browserServiceProcess;
+      browserServiceProcess.stdout.on('data', data => {
```

Comment:
Why don't we switch to `spawn` event and then resolve the spawned process. https://nodejs.org/api/child_process.html#child_process_event_spawn

Reply:
The reason is that the `spawn` event does not guarantee that the browser service has started listening on the port and therefore it add a race condition to the tests.